src/callbacks_step/time_series_dg2d.jl

```diff
@@ -106,8 +154,136 @@ function calc_interpolating_polynomials!(interpolating_polynomials, coordinates,
     return interpolating_polynomials
 end
 
-function calc_interpolating_polynomials(coordinates, element_ids, mesh::TreeMesh, dg,
-                                        cache)
+function calc_interpolating_polynomials!(interpolating_polynomials, coordinates,
+                                         element_ids,
+                                         mesh::UnstructuredMesh2D, dg::DGSEM, cache)
+    @unpack nodes = dg.basis
+
+    wbary = barycentric_weights(nodes)
+
+    # Helper array for a straight-sided quadrilateral element
+    corners = zeros(eltype(mesh.corners), 4, 2)
+
+    for index in 1:length(element_ids)
+        # Construct point
+        x = SVector(ntuple(i -> coordinates[i, index], ndims(mesh)))
+
+        # Convert to unit coordinates; procedure differs for straight-sided
+        # versus curvilinear elements
+        element = element_ids[index]
+        if !mesh.element_is_curved[element]
+            for j in 1:2, i in 1:4
+                # Pull the (x,y) values of the element corners from the global corners array
+                corners[i, j] = mesh.corners[j, mesh.element_node_ids[i, element]]
+            end
+            unit_coordinates = invert_bilinear_interpolation(mesh, x, corners)
+
+            # Sanity check that the computed `unit_coordinates` indeed recover the desired point `x`
+            x_check = straight_side_quad_map(unit_coordinates[1], unit_coordinates[2],
+                                             corners)
+            if !isapprox(x[1], x_check[1], atol = 1e-13) ||
+               !isapprox(x[2], x_check[2], atol = 1e-13)
```

Comment:
Why did you choose these specific values for the absolute tolerance?

Reply:
Wishful think I suppose. I should drop this tolerance to be less strict, maybe around single precision instead

Reply:
I was just curious how you got these values. What if you just use the default tolerance without specifying some?

Reply:
I thought I had to specify tolerances, but it works directly with the default. So, now I just use the defaults.